static/js/humhub/humhub.ui.picker.js

```diff
@@ -332,6 +332,8 @@ humhub.module('ui.picker', function (module, require, $) {
     };
 
     Picker.prototype.prepareItem = function (item) {
+        var that = this;
+        item.id = that.$.data('item-key') === 'id' ? item.id : item.guid;
```

Comment:
@marc-farre These two lines must be replaced with:

```js
const itemKey = this.$.data('item-key');
if (itemKey && itemKey !== 'id' && item.hasOwnProperty(itemKey)) {
    item.id = item[itemKey];
}
```

because I the test [AccountCest](https://github.com/humhub/humhub/actions/runs/14502163811/job/40684141225?pr=7498#step:18:1597) is broken:

![test-user-picker](https://github.com/user-attachments/assets/6f2af81e-00f4-48de-ab2b-1cc454ff0c48)